tests/integration/temporal_client/workflows.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+"""Temporal client sample workflow."""
```

Comment:
Is this sample workflow run just to validate temporal charm is working as expected? Is this testing Airbyte in any way?

Reply:
This runs a sample workflow to test that Temporal is up and running, I've added another test to run an Airbyte sync.